app/code/Magento/Ui/Model/Export/ConvertToCsv.php

```diff
@@ -17,7 +16,7 @@
 class ConvertToCsv
 {
     /**
-     * @var WriteInterface
+     * @var DirectoryList
```

Comment:
Seems like there's some mistake. In the constructor we init `$directory` property as a result of $filesystem->getDirectoryWrite() method, which creates an instance of Magento\Framework\Filesystem\Directory\Write using the corresponding factory. So, basically, it returns `WriteInterface` but not `DirectoryList`. The same situation is for the `ConvertToXml` class.
Please, correct me if I'm wrong.